packages/bpk-docs/package.json

```diff
@@ -83,7 +83,6 @@
     "date-fns": "^1.17.0",
     "es6-promise": "^4.2.4",
     "prop-types": "^15.6.2",
-    "react": "^15.0.0",
```

Comment:
Surely this isn't supposed to have been removed?
Or is is now using react from the main package.json?

Reply:
Yeah it was superfluous, we should have a source of truth for react versions and that is the root `package.json` for web and `native/package.json` for react native.